doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -374,6 +374,25 @@ PSI Variables by Alpha
    The origin-dependence of the CCSD specific rotation in deg/[dm (g/cm^3)]/bohr and the
    length gauge, computed at (x) wavelength, (x) rounded to nearest integer.
 
+.. psivar:: CCname ROOT m CORRELATION ENERGY
+   CCname ROOT m (h) CORRELATION ENERGY
+   CCname ROOT m CORRELATION ENERGY - h TRANSITION
+
+   The correlation energy of given method from ground state to root m in h symmetry.
+   Conventions for root indexing and whether h refers to transition or root
+   irrep are as in :ref:`sec:psivarnotes`.
+
+.. psivar:: CCname ROOT m TOTAL ENERGY
+   CCname ROOT m (h) TOTAL ENERGY
+   CCname ROOT m TOTAL ENERGY - h TRANSITION
+   TD-fctl ROOT m TOTAL ENERGY
+   TD-fctl ROOT m (h) TOTAL ENERGY
+   TD-fctl ROOT m TOTAL ENERGY - h TRANSITION
```

Comment:
For roots, of Total, Correlation, and Excitation Energies, did the last get lost, deferred, or decided against?

Reply:
None of the above. `CC ROOT m EXCITATION ENERGY` did not exist prior to this PR, and I felt adding it would be out of scope of this PR. I agree that it's a bizarre psivar not to have, so I can add it.